services/package.json

```diff
@@ -22,7 +22,7 @@
     "test": "npm run eslint && npm run jest",
     "eslint": "eslint src/. test/. --config .eslintrc.json",
     "start": "node src/",
-    "jest": "jest"
+    "jest": "jest -i --forceExit"
```

Comment:
looks like we might want to file something, or add a `// FIXME`?

Reply:
I didn't bother because I'm not sure if it's worth spending any more time on this. It only manifests when there is no worker pool for `jest` and I sank over four hours into it. I'm done :stuck_out_tongue: